guides/introduction/installation.md

```diff
@@ -304,14 +309,29 @@ The change is to define the `live_view` and `live_component` functions in your `
   end
 ```
 
-Note that LiveViews are automatically configured to use a "live.html.heex" layout in this line:
+Note that LiveViews are automatically configured to use a `live.html.heex` layout in this line:
 
 ```elixir
 use Phoenix.LiveView,
   layout: {MyAppWeb.LayoutView, "live.html"}
 ```
 
-"layouts/root.html.heex" is shared by regular and live views, "app.html.heex" is rendered inside the root layout for regular views, and "live.html.heex" is rendered inside the root layout for LiveViews. "live.html.heex" typically starts out as a copy of "app.html.heex", but using the `@socket` assign instead of `@conn`. Check the [Live Layouts](live-layouts.md) guide for more information.
+`layouts/root.html.heex` is shared by regular and live views, `app.html.heex` is rendered inside the root layout for regular views, and `live.html.heex` is rendered inside the root layout for LiveViews. `live.html.heex` typically starts out as a copy of `app.html.heex`, but using the `@socket` assign instead of `@conn`. Check the [Live Layouts](live-layouts.md) guide for more information.
+
+## LiveView life-cycle event logging
+
+If you would like logs emitted for supported LiveView life-cycle events, you can add the following line to your `lib/my_app/application.ex` in the `c:start/2` function:
+
+```elixir
+# lib/my_app/application.ex
+
+@impl true
+def start(_type, _args) do
+  # ...
+  Phoenix.LiveView.Logger.install()
```

Comment:
we should do this by default in LV's start, and we could make it configurable like phoenix https://github.com/phoenixframework/phoenix/blob/454b1b74741b77d179a24f82431e57e0996a17d0/lib/phoenix.ex#L63-L65

Reply:
I agree! Where should I put the call to `Phoenix.LiveView.Logger.install/0`? I don't see a `start/2` defined for Live View itself: should this be a conditional call in `Phoenix.start/2` if the `Phoenix.LiveView.Logger` module is defined?